orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -318,6 +318,12 @@ def _get_session_config(self):
         session_options.execution_order = onnxruntime.ExecutionOrder.PRIORITY_BASED
         # 0:Verbose, 1:Info, 2:Warning. 3:Error, 4:Fatal. Default is 2.
         session_options.log_severity_level = int(self._debug_options.logging.log_level)
+        # Disable memory alleviation by default. Allow user to enable it via environment variable.
+        enable_memory_alleviation = ortmodule._defined_from_envvar(
+            "ORTMODULE_ENABLE_MEMORY_ALLEVIATION", "Dropout:0,Gelu:0,Tile:0", warn=True
+        )
+        warnings.warn(f"ORTMODULE_ENABLE_MEMORY_ALLEVIATION user config: {enable_memory_alleviation}")
+        session_options.add_session_config_entry("optimization.enable_memory_alleviation", enable_memory_alleviation)
```

Comment:
Oh, you mean "session_options.add_session_config_entry("optimization.enable_memory_alleviation", right?

I see other code doing this way, I think this might not be needed so far, maybe we can have an item to revisit how to expose and use those options.